apps/session-manager/src/app.ts

```diff
@@ -191,6 +194,24 @@ export const newApp: (
     LOLLIPOP_CLIENT,
   );
 
+  app.post(
+    `${ZENDESK_BASE_PATH}/jwt`,
+    authMiddlewares.bearerZendesk,
+    pipe(
```

Comment:
I was checking https://github.com/pagopa/io-infra/pull/992 and I realized that IP filtering is missing, along with its env variable.
Can we add in this PR? We will need trick the list for making an e2e test, but I think it will be more consistent

Reply:
added in https://github.com/pagopa/io-auth-n-identity-domain/pull/34/commits/5ce01defe6bac2f60b8c627a15ead3ba50a1fbd4